framework/material/Dialog.kt

```diff
@@ -79,4 +82,14 @@ object Dialog {
             }
         }
     }
+
+    @Composable
+    fun RowSpacer() {
```

Comment:
I feel like these aren't really spacers that we use in the conventional sense.

`RowSpacer` exists to ensure that multiple **items** in a horizontal **list** (`Row`) have consistent **spacing** across the application. Whereas the consistent style we're trying to enforce in dialogs is more to do with the **margin** around the dialog content (or equivalently, and perhaps more clearly, the **padding** of the dialog).

As such, I'd suggest extracting `Dialog's` padding as a variable if it isn't yet, and then using the `padding` modifier to apply it to the preference dialog's action buttons area.

(Note that with the current model we end up with both `Dialog.RowSpacer` and `Form.RowSpacer` in the same file and that's a little odd - not a killer, but just a sign that something's afoot.)

Reply:
Done.